client/my-sites/plans/jetpack-plans/selector.tsx

```diff
@@ -69,6 +69,20 @@ const SelectorPage: React.FC< SelectorPageProps > = ( {
 		);
 	}, [ dispatch, rootUrl, siteSlug, viewTrackerPath ] );
 
+	const { unlinked, purchasetoken, purchaseNonce, site } = urlQueryArgs;
+	const canDoSiteOnlyCheckout = unlinked && !! site && ( !! purchasetoken || purchaseNonce );
```

Comment:
Although I think the condition will still "work" the way you have it, I think you meant to have:
`&& !! ( purchasetoken || purchaseNonce )` 
as opposed to:
`&& ( !! purchasetoken || purchaseNonce )`
```suggestion
	const canDoSiteOnlyCheckout = unlinked && !! site && !! ( purchasetoken || purchaseNonce );
```

Reply:
Good catch, Bryan!